### PR TITLE
SFMPlus Add Feature Actuator (1)

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp [hunk 1]
```diff
@@ -37,6 +37,6 @@
 
 #define HDG_HOLD_SPEED_SWITCH_DECEL   15.433  //30kts GS
 #define HDG_HOLD_SPEED_SWITCH_ACCEL   20.577  //40kts GS
-#define HDG_HOLD_BREAKOUT_VALUE       0.03
-#define VEL_HOLD_BREAKOUT_VALUE       0.06
-#define ATT_HOLD_BREAKOUT_VALUE       0.09
\ No newline at end of file
+#define HDG_HOLD_BREAKOUT_VALUE       0.030
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp:40]: Why did you add the suffixing 0?

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getRtrRPM.sqf [hunk 2]
```diff
@@ -18,8 +18,8 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-private _cfg         = configOf _heli;
-private _flightModel = getText (_cfg >> "fza_flightModel");
+private _configVehicles = configFile >> "CfgVehicles" >> typeof _heli;
```
[the remark this answers is not in the document]
Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getRtrRPM.sqf:21]: As far as I can tell all of these changes are redundant and could be omitted?

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/headers/systemConstants.h [hunk 3]
```diff
@@ -51,4 +51,21 @@
 #define ASE_AUTOPAGE_OFF 0
 //--Chaff dispenser
 #define ASE_CHAFF_STATE_SAFE 0
-#define ASE_CHAFF_STATE_ARM  1
\ No newline at end of file
+#define ASE_CHAFF_STATE_ARM  1
+
+//FMC
+//--FMC Pitch
+#define FMC_PITCH_OFF 0
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/headers/systemConstants.h:58]: Are these used at all?

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_sign.sqf [hunk 4]
```diff
@@ -1,13 +1,18 @@
 /* ----------------------------------------------------------------------------
 Function: fza_fnc_sign
+
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_sign.sqf:3]: Are these changes required?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHeadingHold.sqf [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
-private _desiredHdg = _heli getVariable "fza_ah64_hdgHoldDesiredHdg";
-private _hdgError   = [_curHdg - _desiredHdg] call CBA_fnc_simplifyAngle180;
-private _curVel     = vectorMagnitude [velocityModelSpace _heli # 0, velocityModelSpace _heli # 1];
-private _subMode    = _heli getVariable "fza_ah64_hdgHoldSubMode";
-private _attSubMode = _heli getVariable "fza_ah64_attHoldSubMode";
-private _output     = 0.0;
+private _pidHdg      = _heli getVariable "fza_sfmplus_pid_hdgHold";
+//_pidHdg set ["kp", H_KP];
+//_pidHdg set ["ki", H_KI];
+//_pidHdg set ["kd", H_KD];
```
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHeadingHold.sqf:7]: Complete

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getRtrRPM.sqf (hunk 2 above)
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getRtrRPM.sqf:21]: Fixed

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp:40]: 3 digit precision was used to allow for 2.5%, was updated to 3, the 3 digit precision remained

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/headers/systemConstants.h (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/headers/systemConstants.h:58]: Nope, left overs. Removed

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_sign.sqf (hunk 4 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_sign.sqf:3]: Keeps the documentation formatting inline with the rest of the documentation formatting.

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageACUtilHandleControl.sqf [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        _heli setVariable ["fza_ah64_fmcYawOn", _newState];
+    };
+    //Coll
+    case "l4": {
+        private _newState = [true, false] select (_heli getVariable "fza_ah64_fmcCollOn" == true);
+        _heli setVariable ["fza_ah64_fmcCollOn", _newState];
+    };
+    //Trim
+    case "l5": {
+        private _newState = [true, false] select (_heli getVariable "fza_ah64_fmcTrimOn" == true);
```
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageACUtilHandleControl.sqf]: Already resolved

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/XEH_preInit.sqf [hunk 7]
```diff
@@ -69,6 +69,15 @@ private _projName = "AH-64D Official Project";
     0
 ] call CBA_fnc_addSetting;
 
+[
+    "fza_ah64_sfmplusEnableTorqueSim",
+    "CHECKBOX",
+    ["Main Rotor Torque Simulation", "Turns on/off main rotor torque simulation. Meant for users who fly mouse & keyboard."],
```
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/XEH_preInit.sqf:75]: Made it more specific.

**Comment by mattysmith22** [comment]
These changes were split out and merged as a series of other PRs
